frontends/p4/typeChecking/typeChecker.cpp

```diff
@@ -3179,6 +3179,10 @@ const IR::Node* TypeInference::postorder(IR::MethodCallExpression* expression) {
         auto prop = findContext<IR::Property>();
         if (prop != nullptr && prop->name == IR::TableProperties::actionsPropertyName)
             inActionsList = true;
+        if (findContext<IR::Key>()) {
```

Comment:
We should probably bring this up again in the language design group. (or figure out what the conclusion was when we discussed this last time.)

Reply:
I will remove this code, it is not necessary, since actions cannot be called in a key computation because they return void, so they cannot be part of an expression anyway.